app/services/draft_asset_cleanup_service.rb

```diff
@@ -7,8 +7,14 @@ def call(edition)
 
     return unless previous_revision
 
-    current_assets = current_revision.image_revisions.flat_map(&:assets)
-    previous_assets = previous_revision.image_revisions.flat_map(&:assets)
+    current_image_assets = current_revision.image_revisions.flat_map(&:assets)
```

Comment:
Could we extract this into `current_assets` and `previous_assets` methods (or something similar) to keep `#call` a bit shorter?

Reply:
Yep that is a good idea!